api/v1beta1/foundationdbcluster_types.go

```diff
@@ -471,15 +471,42 @@ type ProcessGroupStatus struct {
 	Addresses []string `json:"addresses,omitempty"`
 	// Remove defines if the process group is marked for removal.
 	Remove bool `json:"remove,omitempty"`
+	// RemoveTimestamp if not empty defines when the process group was marked for removal.
+	RemoveTimestamp metav1.Time `json:"removeTimestamp,omitempty"`
 	// Excluded defines if the process group has been fully excluded.
 	// This is only used within the reconciliation process, and should not be considered authoritative.
 	Excluded bool `json:"excluded,omitempty"`
+	// ExcludedTimestamp defines when the process group has been fully excluded.
+	// This is only used within the reconciliation process, and should not be considered authoritative.
+	ExcludedTimestamp metav1.Time `json:"excludedTimestamp,omitempty"`
```

Comment:
There is an issue in Kubernetes version < 1.20 (see: https://github.com/kubernetes/kubernetes/issues/86811). Since 1.19 is not "supported" anymore we might think about dropping the test case for 1.19 (https://kubernetes.io/releases/). Another option would be to make this `*metav1.Time` with all the additional `nil` checks which should be fine since we never use that field directly (only over the according methods).